README.md

```diff
@@ -33,7 +38,12 @@ evaluation outputs.
 
 To change these default directories simply run `armory configure` after installation.
 
-If installing from the git repo in editable mode, ensure that your pip version is 22+. 
+If installing from the git repo in editable mode, ensure that your pip version is 22+.
+
+## `docker`
```

Comment:
this docker section seems like it could either be deleted or fleshed out in more detail